src/posts/components/Post.tsx

```diff
@@ -0,0 +1,70 @@
+import "../../assets/styles/responsive.scss"; // تأكد من المسار الصحيح
+
+import React from "react";
+
+interface PostProps {
```

Comment:
in this component props, you can pass `Post` object instead of passings its properties, this insures more scalability to the code